lib/CXGN/Phenotypes/StorePhenotypes.pm

```diff
@@ -557,6 +566,21 @@ sub store {
     return ($error_message, $success_message, \@stored_details);
 }
 
+sub store_stock_note {
+    my $self = shift;
+    my $stock_id = shift;
+    my $note_array = shift;
+    my $operator = shift;
+    my $note = $note_array->[0];
+    my $timestamp = $note_array->[1];
+    $operator = $note_array->[2] ? $note_array->[2] : $operator;
+
+    print STDERR "Stock_id is $stock_id and note in sub is $note, timestamp is $timestamp, operator is $operator\n";
+
+    $note = $note ." (Operator: $operator, Time: $timestamp)";
+    my $stock = $self->bcs_schema()->resultset("Stock::Stock")->find( { stock_id => $stock_id } );
+    $stock->create_stockprops( { 'notes' => $note }, { autocreate => 1 } );
```

Comment:
i would avoid using the autocreate here. you could add a db patch for 'notes' as 'stockprop'

Reply:
right, i wouldn't advocate for the use of that there! i think we decided a long time ago that cvterms would only be created via db patches and not in the code

Reply:
Haha ok guess I wasn't thinking about that and took a page out of the wrong book. Or a line out of the wrong module.

The notes stockprop is added in db patch 88, so I just removed the unnecessary autocreate. We should probably should refactor how add stockprops is handled on stock detail pages but not trying to tackle that here.